pkg/metareconciler/clusterducktype/testdata/features/smoke.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Reconcile ClusterDuckType
```

Comment:
I must say that I don't have a clue on how I would go about constructing a new test and what the keywords are, which matter, etc. So, I really can't comment on these.

I'm curious how error prone these are. For example, were I to change the last line from '1 controllers' to '1 controller', would that work? If not, what would the error behaviour be. Not necessarily something that I expect an answer in this comment, just jotting it down because I'm not familiar with this enough to say anything except I find this a bit confusing :)